man/journal-upload.conf.xml

```diff
@@ -129,6 +129,28 @@
         <listitem><para>Takes a boolean value, enforces using compression without content encoding negotiation.
         Defaults to <literal>false</literal>.</para>
 
+        <xi:include href="version-info.xml" xpointer="v258"/></listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>Header=</varname></term>
+
+        <listitem><para>Specifies an additional HTTP header to be added to each request to a URL.
+        Takes a pair of header name and value separated with a colon(<literal>:</literal>),
+        e.g. <literal>Name:Value</literal>.
+        Header name can contain alphanumeric values, "_" and "-" symbols additionally.
```

Comment:
Please use `<literal>` tag for quotation: 
`<literal>_</literal> and <literal>-</literal>`